pkg/kubectl/rollback.go

```diff
@@ -338,13 +339,39 @@ func (r *StatefulSetRollbacker) Rollback(obj runtime.Object, updatedAnnotations
 	}
 
 	// Restore revision
-	if _, err = r.c.AppsV1beta1().StatefulSets(sts.Namespace).Patch(sts.Name, types.StrategicMergePatchType, toHistory.Data.Raw); err != nil {
+	if _, err = r.c.AppsV1().StatefulSets(sts.Namespace).Patch(sts.Name, types.StrategicMergePatchType, toHistory.Data.Raw); err != nil {
 		return "", fmt.Errorf("failed restoring revision %d: %v", toRevision, err)
 	}
 
 	return rollbackSuccess, nil
 }
 
+// findHistoryV1 returns a controllerrevision of a specific revision from the given controllerrevisions.
+// It returns nil if no such controllerrevision exists.
+// If toRevision is 0, the last previously used history is returned.
```

Comment:
Add a TODO here to rename this to findHistory when other controllers have been upgraded.